crates/apollo-mcp-server/src/runtime/trace.rs

```diff
@@ -0,0 +1,111 @@
+use opentelemetry::{KeyValue, global, trace::TracerProvider as _};
+use opentelemetry_sdk::{
+    Resource,
+    metrics::{MeterProviderBuilder, PeriodicReader, SdkMeterProvider},
+    trace::{RandomIdGenerator, SdkTracerProvider},
+};
+use opentelemetry_semantic_conventions::{
+    SCHEMA_URL,
+    attribute::{DEPLOYMENT_ENVIRONMENT_NAME, SERVICE_VERSION},
+};
+use tracing_opentelemetry::{MetricsLayer, OpenTelemetryLayer};
+use tracing_subscriber::{layer::SubscriberExt, util::SubscriberInitExt};
+
+use crate::runtime::{Config, logging::Logging};
```

Comment:
nit: Would be helpful to use ‎`super` instead to communicate that they come from the parent module?

```suggestion
use super::{Config, logging::Logging};
```